packages/vue/generators.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Nx Vue",
+  "version": "0.1",
+  "schematics": {
```

Comment:
Let's remove the `schematics` field. We won't support running `@nx/vue` from Angular CLI.

The latest commit here does not include Angular CLI fields. https://github.com/nrwl/nx/pull/19009